src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user?: User;
+};
+
+export const UserInfo: FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has `null` or `undefined` value. It's better to handle the condition outside of the component and render `UserInfo` only if `user` is not `null` or `undefined`. This way, you avoid rendering an anchor tag with a `mailto:` link that doesn't contain an email address, which could be confusing or misleading to users.

Reply:
yep, good point 🔥 
let's create an additional condition before rendering